From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aswin-mcw <aswin.kalies@multicorewareinc.com>
Date: Wed, 9 Sep 2026 13:14:27 +0530
Subject: [PATCH 048/109] remove dll name

(cherry picked from commit cd09e46caa504ce94cd1fd629309a8ca9010ea24)
---
 winsup/cygwin/cygwin.din | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/winsup/cygwin/cygwin.din b/winsup/cygwin/cygwin.din
index 073b8d07f..675678180 100644
--- a/winsup/cygwin/cygwin.din
+++ b/winsup/cygwin/cygwin.din
@@ -1,6 +1,3 @@
-LIBRARY "msys-2.0.dll" BASE=0x180040000
-
-EXPORTS
 # Exported variables
 __argc DATA
 __argv DATA
